plugins/module_utils/linode_common.py

```diff
@@ -80,6 +80,10 @@
         "doc_hide": True,
         "fallback": (env_fallback, ["LINODE_UA_PREFIX"]),
     },
+    "ca_path": {
+        "type": "str",
+        "description": "A path to a custom certificate authority for using alternate APIs.",
```

Comment:
Could there also be an environment variable fallback like we do for `linode_token`? e.g. `LINODE_CA`